suites/manage/ocp/test_ocs_284.py

```diff
@@ -0,0 +1,138 @@
+import os
+import logging
+import ocs.defaults as defaults
+from ocs.ocp import OCP
+from utility import utils, templating
+from ocsci.enums import TestStatus
+
+log = logging.getLogger(__name__)
+
+RBD_SC_YAML = os.path.join("ocs-deployment", "storage-manifest.yaml")
+TEMP_SC_YAML_FILE = '/tmp/tmp-storage-manifest.yaml'
+
+RBD_PVC_YAML = os.path.join("ocs-deployment", "PersistentVolumeClaim.yaml")
+TEMP_PVC_YAML_FILE = '/tmp/tmp-persistentVolumeClaim.yaml'
+
+occli = OCP(kind='service', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+
+
+def create_rbd_cephpool(poolname, storageclassname):
+    """
+    Creates rbd storage class and ceph pool
```

Comment:
Please add a description to the function parameters

Reply:
> Please add a description to the function parameters

Done